packages/cli/src/config/settings.ts

```diff
@@ -61,6 +69,19 @@ export interface AccessibilitySettings {
   screenReader?: boolean;
 }
 
+export interface DisplaySettings {
```

Comment:
this merge didn't go correctly. Please update settingSchema.ts instead of add your additional option. Please also align with additional naming for settings. there are similar existing settings for hiding other portions of the ui.